packages/web-react/src/components/Collapse/demo/CollapseHideTrigger.tsx

```diff
@@ -17,7 +17,7 @@ const CollapseHideTrigger = () => {
       <a href="#" role="button" onClick={toggleHandler} aria-expanded={isOpen}>
         <span className="accessibility-closed">… more</span>
       </a>
```

Comment:
Can we use a button that is semantically more appropriate than a link?


```suggestion
<button type="button" onClick={toggleHandler} aria-expanded={isOpen} className="button-unstyled">
<span className="accessibility-closed">… more</span>
</a>
```

Reply:
And another idea: add a demo with more & less buttons as well?